### PR TITLE
fix: dont mark version as dirty temporarily and investigate git status

### PR DESCRIPTION
**Pull request by gak**

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -51,6 +51,8 @@ build +tools: build-protos build-zips build-frontend
   #!/bin/bash
   shopt -s extglob
 
+  git diff
+
```
Comment (wesbillman) [inline Justfile:55]: Is this for logging @gak?

File: backend/controller/cronjobs/sql/models.go [hunk 2]
```diff
@@ -478,6 +478,14 @@ type ModuleConfiguration struct {
 	Value     []byte
 }
 
+type ModuleSecret struct {
+	ID        int64
+	CreatedAt time.Time
+	Module    optional.Option[string]
+	Name      string
+	Url       string
+}
```
Comment (wesbillman) [inline backend/controller/cronjobs/sql/models.go:487]: I'm guessing this is unrelated, but needed to be checked in?

**Review by wesbillman — APPROVED** [review]

File: Justfile (hunk 1 above)
Comment (gak) [reply Justfile:55]: Yes, I need to see what is causing the repo to be dirty.

File: backend/controller/cronjobs/sql/models.go (hunk 2 above)
Comment (gak) [reply backend/controller/cronjobs/sql/models.go:487]: It is related to code generated in the CI causing the repo to be dirty. I'm under the impression these sqlc files needed to be checked in?

Comment (deniseli) [reply backend/controller/cronjobs/sql/models.go:487]: Your impression is correct :) Looks like whoever added ModuleSecret to the db schema just didn't know they needed to run `just build-sqlc`.

**Pull request merged**